### PR TITLE
Use navigator.language before navigator.languages[0]

### LINKED ISSUE #6: Chrome browser language not necessarily first in languages list
It seems that in the latest version of Chrome (v. 59.0.3071.115), the display language is not necessarily the first one in the _window.navigator.languages_ list (so taking languages[0] gets a wrong result).

See screenshots below:

**Chrome navigator**
chrome_navigator

**Chrome settings**
chrome_settings

Maybe it would be an idea to take into account the _window.navigator.language_ value first and, if not found, use the first entry in the _window.navigator.languages_ array.

P.S.: As far as I could see, now even the latest versions of IE and Edge set the _window.navigator.language_ value.

### PR DESCRIPTION
**Pull request by R4DIC4L**
Chrome browser language not necessarily first in languages list
Closes #6

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-    // latest versions of Chrome, Firefox, and Safari set this correctly
-    lang = navigator.language
+    lang = navigator.userLanguage;
+  } else if (navigator.language) {
+    // latest versions of Chrome, Firefox, Edge and Safari set this correctly
+    lang = navigator.language;
+  } else if (navigator.languages && navigator.languages.length) {
+    // latest versions of Chrome and Firefox set this correctly
+    lang = navigator.languages[0];
+  } else if (navigator.browserLanguage) {
```
Comment (lnicola) [inline index.js]: Should this be moved between `language` and `languages`?

Comment (R4DIC4L) [reply index.js]: Changed the order so that it first checks for `browserLanguage` for Edge

File: index.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+  if (navigator.userLanguage) {
     // IE only
-    lang = navigator.userLanguage
-  } else {
-    // latest versions of Chrome, Firefox, and Safari set this correctly
-    lang = navigator.language
+    lang = navigator.userLanguage;
+  } else if (navigator.language) {
+    // latest versions of Chrome, Firefox, Edge and Safari set this correctly
+    lang = navigator.language;
```
Comment (WilliamHolmes) [inline index.js:9]: Wont this return the browsers install language and not the locale?
ie. Set your browser to a language other than the install language and try this out. (Chrome)
image
update the browser language settings ...
image

**Review by WilliamHolmes — COMMENTED** [review]
@R4DIC4L I dont think this will return the expected locale.

navigator.language returns the browser install language (at least in Chrome)

File: index.js (hunk 2 above)
Comment (R4DIC4L) [reply index.js:9]: What Chrome version are you using? In my browser, the languages order does not change (I think it's the one in the list, and I make it default / show browser in language without reordering them). See screenshots below:

This is how I set it:
chrome_language_set_default_es

This is what it looks like:
chrome_language change_es

Also, in Firefox, Edge and IE, I have no languages in the navigator.

Comment (lnicola) [reply index.js:9]: On Linux, `navigator.language` is the locale (as in `LC_ALL`) and `navigator.languages` is the language list form Chrome's settings. I'm not sure about windows.

**Pull request closed without merging**